tests/spdx/writer/tagvalue/test_annotation_writer.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2023 spdx contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from unittest.mock import MagicMock, call, mock_open, patch
+
+from spdx.writer.tagvalue.annotation_writer import write_annotation
+from tests.spdx.fixtures import annotation_fixture
+
+
+def test_annotation_writer():
+    annotation = annotation_fixture()
+
+    mock: MagicMock = mock_open()
+    with patch(f"{__name__}.open", mock, create=True):
+        with open("foo", "w") as file:
+            write_annotation(annotation, file)
+
+    mock.assert_called_once_with("foo", "w")
+    handle = mock()
+    handle.write.assert_has_calls(
```

Comment:
This method allows other calls before or after the given list of calls. I haven't found a way to check that there are no such calls but maybe this isn't too important to check right now, anyway.

Reply:
Yes, another option would be `assert_has_calls` but this wouldn't check the order of calls, which is more important I think than the "risk" of calls before or after this statement.